src/components/Charts/ModelChart.js

```diff
@@ -205,7 +205,7 @@ const ModelChart = ({
         if (this.series.userOptions.type === 'line') {
```

Comment:
i think this is actually the bug we should check for the name instead of the line type since the line changed to a spline hospitilzations is correct below but the spline change broke

Reply:
Can you clarify your bug report then?  The bug report only alludes to the typo. Is there something I'm missing?

Reply:
yup the bug is that for the available bed line, the tooltip is displaying hospitilizations, when it should in-fact be showing expected beds

Reply:
Someone changed the series type a couple days ago from line to spline which is why the available beds series is now showing hospitalizations